pkg/logging/logger.go

```diff
@@ -193,3 +200,14 @@ func (l *LoggerImpl) Debugf(template string, args ...interface{}) {
 func (l *LoggerImpl) Debugw(msg string, keysAndValues ...interface{}) {
 	l.InnerLogger.Debugw(msg, keysAndValues...)
 }
+
+func (l *LoggerImpl) processStructuredLog(msg string, keysAndValues ...interface{}) {
```

Comment:
Is this function intended to do something other than create notifications? If not maybe a more concrete name like `createNotification` or `notifyFromLog` or something like that would be better suited.